package.json

```diff
@@ -45,11 +46,13 @@
         "proof-of-membership",
         "monorepo"
     ],
-    "workspaces": [
-        "apps/*",
-        "packages/*",
-        "packages/contracts/contracts"
-    ],
+    "workspaces": {
```

Comment:
Could you keep this section as it was before?

Reply:
Sure done that change @vplasencia. Thanks.